internal/status_test.go

```diff
@@ -0,0 +1,47 @@
+package internal
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/cisco-open/grabit/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetStatusString(t *testing.T) {
+	// Test 1000 resources.
+	content := `abcdef`
+	port := test.TestHttpHandler(content, t)
+	resources := []Resource{}
+	algo := "sha256"
+	for i := 0; i < 1000; i++ {
+		resource := Resource{Urls: []string{fmt.Sprintf("http://localhost:%d/test%d.html", port, i)}, Integrity: fmt.Sprintf("%s-vvV+x/U6bUC+tkCngKY5yDvCmsipgW8fxsXG3Nk8RyE=", algo), Tags: []string{}, Filename: ""}
+		resources = append(resources, resource)
+	}
+	ctx, _ := context.WithCancel(context.Background())
+
+	// Test resource generation.
+	assert.Len(t, resources, 1000)
```

Comment:
You can remove this. I just added it when I gave you this code to show what it was doing, but there is no need to test our own pre-test code.